iceoryx_posh/test/moduletests/test_roudi_cmd_line_parser_config_file_option.cpp

```diff
@@ -97,10 +103,65 @@ TEST_F(CmdLineParserConfigFileOption_test, HelpLongOptionLeadsProgrammNotRunning
     args[0] = &appName[0];
     args[1] = &option[0];
 
-    iox::config::CmdLineParserConfigFileOption sut;
+    CmdLineParserConfigFileOption sut;
     auto result = sut.parse(numberOfArgs, args);
 
-    EXPECT_THAT(result.has_error(), Eq(false));
-    EXPECT_THAT(result.value().run, Eq(false));
+    EXPECT_FALSE(result.has_error());
+    EXPECT_FALSE(result.value().run);
 }
+
+TEST_F(CmdLineParserConfigFileOption_test, WrongOptionLeadsUnkownOptionResult)
+{
+    constexpr uint8_t numberOfArgs{2U};
+    char* args[numberOfArgs];
+    char appName[] = "./foo";
+    char option[] = "--unknown";
```

Comment:
for better readability I would rename the variable from `option` into `unknownOption`.

Reply:
Looks like it's done.